lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -855,6 +855,140 @@ static void optimizeBatchNorm(Function *F) {
   } // For all nodes in the graph.
 }
 
+/// Check if the \p inputs of this method (vector of NodeValues) can be
```

Comment:
The name of the function does not make sense. Are we checking that some Concats are consecutive? We have a function that does that. Do we need to refactor it to support this usecase?